README.md

```diff
@@ -221,7 +240,9 @@ dependencies {
 }
 ```
 
-Validating your produced values should feel familiar to those who have used [Turbine](https://github.com/cashapp/turbine/).
+Unlike `moleculeFlow(clock = Immediate)`, `testMolecule` simulates clock ticks:
+Calling `awaitItem` yields to running coroutines and "ticks" the Molecule's clock until a change to snapshot state occurs, triggering a fresh recomposition and a new item.
+If snapshot state is never changed, no recomposition will ever occur and `awaitItem` will fail.
```

Comment:
I think this is a pretty good explanation, but also gets at my discomfort with the approach: it's a push approach. I've found that attempting to "push" coroutines code (under test or in prod) is like trying to pushing a wet noodle across a dinner plate